docs/source/concepts/ledger-model/ledger-integrity.rst

```diff
@@ -410,7 +410,7 @@ agree to this obligation. In other words, the painter did not *authorize*
 the creation of the obligation.
 
 In a DA ledger, a party can **authorize** a subaction of a commit in
-one of the two ways:
+either of the following way:
```

Comment:
"Ways" was correct.

```suggestion
either of the following ways:
```

Reply:
Thank you for your input -- changed accordingly.